examples/supported-python-versions-example/control-service-setup/README.md

```diff
@@ -0,0 +1,258 @@
+                  image: busybox
+                  imagePullPolicy: IfNotPresent
+              restartPolicy: OnFailure
+          ttlSecondsAfterFinished: 600
+
+
+deploymentSupportedPythonVersions:
+  release:
+    baseImage: "registry.hub.docker.com/versatiledatakit/data-job-base-python-3.9:latest"
+    vdkImage: "registry.hub.docker.com/versatiledatakit/quickstart-vdk:release"
+  pre-release:
+    baseImage: "registry.hub.docker.com/versatiledatakit/data-job-base-python-3.9:latest"
+    vdkImage: "registry.hub.docker.com/versatiledatakit/quickstart-vdk:pre-release"
+
+deploymentDefaultPythonVersion: "pre-release"
+```
+</details>
+
+Most of the content of the file are configurations necessary for the Control Service deployment to operate as required
+by the vdk server (to properly connect to the git server and the image registry instance). The change relevant to this
+example is:
+
+```yaml
+deploymentSupportedPythonVersions:
+  3.9-release:
+    baseImage: "registry.hub.docker.com/versatiledatakit/data-job-base-python-3.9:latest"
+    vdkImage: "registry.hub.docker.com/versatiledatakit/quickstart-vdk:release"
+  3.9-pre-release:
+    baseImage: "registry.hub.docker.com/versatiledatakit/data-job-base-python-3.9:latest"
+    vdkImage: "registry.hub.docker.com/versatiledatakit/quickstart-vdk:pre-release"
+
+deploymentDefaultPythonVersion: "pre-release"
+```
+
+As can be seen in the code snippet, we set a `deploymentSupportedPythonVersions` property as a YAML map, which will be
+converted to a JSON string when the service is deployed by helm. We set two python versions, that both use
+`data-job-base-python-3.9:latest` as a __baseImage__ (the data job base image used to build the data job image), and
+the `quickstart-vdk:pre-release` and `quickstart-vdk:release` images for the __vdkImage__ instances (VDK SDK images
+used during data job executions).
+
+Service Helm Upgrade
+--------------------
+
+If you have executed `vdk server --install`, as prompted at the beginning of this example, the only thing left is to
+upgrade the Control Service to adopt the new configuration. To do that, open a terminal and save the above long yaml
+script into a file named __values.yaml__, or navigate to the example directory.
+
+Once ready, activate the python virtual environment where you installed quickstart-vdk, if you haven't done so already,
+and execute:
+
+```shell
+helm upgrade vdk vdk-gitlab/pipelines-control-service -f values.yaml
+```
+
+After the execution of the above command, wait for a few minutes for the service to be updated. To check if the new
+configuration has been applied, you can either check the kubernetes cluster by running
+```shell
+kubectl get pods
+```
+```shell
+kubectl describe pod <pod-name-of-the-vdk-service-deployment>
+```
+and look for an environment variable named `DATAJOBS_DEPLOYMENT_SUPPORTED_PYTHON_VERSIONS`, or
+open a new terminal and run:
+```shell
+kubectl port-forward service/vdk-svc 8092:8092
```

Comment:
http://localhost:8092 should be accessible upon vdk server installation, without an explicit port forward

Reply:
It is, I just had issues with vdk-server, which are now resolved, so I've removed the additional commands. Thanks